addons/website/tests/test_create_website.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import patch
+from odoo.tests import HttpCase
+import odoo.tests
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestCreateWebsite(HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        # Actions are quite tricky in this flow.
+        # Because we cannot do _theme_upgrade_upstream
+        # the theme install action isn't consummed
+        # and it puts the client back to choosing a theme
+        # once it just picked one. So we make sure only the
+        # usual action is enabled
+        actions = self.env['ir.actions.todo'].search([('state', '=', 'open')])
+        actions.write({'state': 'done'})
+        self.env['ir.actions.todo'].create({'action_id': self.env.ref('website.action_website_edit').id, 'state': 'open'})
+
+    def setUp(self):
+        super().setUp()
+        patcher = patch('odoo.addons.website.models.ir_module_module.IrModuleModule._theme_upgrade_upstream', wraps=self._theme_upgrade_upstream)
+        patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def test_01_create_website(self):
+        self.env['res.lang'].create({
+            'name': 'Parseltongue',
+            'code': 'pa_GB',
+            'iso_code': 'pa_GB',
+            'url_code': 'pa_GB',
+        })
+        self.start_tour('/', 'test_create_website_1', login='admin')
```

Comment:
Note that the test name is confusing, this is not at all about creating a website but just selecting a new lang?